open_academy/views/course.xml

```diff
@@ -9,9 +9,11 @@
             <group>
               <field name="title"/>
             </group>
-            <group>
-              <field name="description"/>
-            </group>
+            <notebook>
+              <sheet>
```

Comment:
change sheet tag to page tag, sheet it's to draw the following spaces (in red)
![image](https://user-images.githubusercontent.com/47753811/148996702-2935588f-8148-4ad7-830b-2a54d6ff3b82.png)

Please change to
```suggestion
              <page>
```